wpilibjIntegrationTests/build.gradle

```diff
@@ -24,8 +24,11 @@ dependencies {
     compile 'com.googlecode.junit-toolbox:junit-toolbox:2.0'
     compile 'org.apache.ant:ant:1.9.4'
     compile 'org.apache.ant:ant-junit:1.9.4'
+    compile files(wpilibj.sourceSets.test.output.classesDir)
 }
 
+compileJava.dependsOn tasks.getByPath(':wpilibj:testClasses')
```

Comment:
so, what does this do? Or rather, how was it working before? I see the test jar is dependant on wpilibj jar but not sure if that enforces wpilibj to compile before tests do